src/include/migraphx/register_target.hpp

```diff
@@ -27,20 +27,35 @@
 #include <migraphx/config.hpp>
 #include <migraphx/target.hpp>
 #include <migraphx/auto_register.hpp>
+#include <migraphx/dynamic_loader.hpp>
 #include <cstring>
 #include <vector>
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
+void store_target_lib(const dynamic_loader& lib);
+
+void target_map_init();
```

Comment:
Should be called `register_target_init`.

Reply:
Done.